client/state/action-types.js

```diff
@@ -36,6 +36,10 @@ export const ANALYTICS_STAT_BUMP = 'ANALYTICS_STAT_BUMP';
 export const AUTOMATED_TRANSFER_ELIGIBILITY_REQUEST = 'AUTOMATED_TRANSFER_ELIGIBILITY_REQUEST';
 export const AUTOMATED_TRANSFER_ELIGIBILITY_UPDATE = 'AUTOMATED_TRANSFER_ELIGIBILITY_UPDATE';
 export const AUTOMATED_TRANSFER_STATUS_SET = 'AUTOMATED_TRANSFER_STATUS_SET';
+export const BILLING_DATA_RECEIVE = 'BILLING_DATA_RECEIVE';
+export const BILLING_DATA_REQUEST = 'BILLING_DATA_REQUEST';
+export const BILLING_DATA_REQUEST_FAILURE = 'BILLING_DATA_REQUEST_FAILURE';
+export const BILLING_DATA_REQUEST_SUCCESS = 'BILLING_DATA_REQUEST_SUCCESS';
```

Comment:
The `data` part here seems a bit redundant. Maybe `billing-history` is a better fit to differentiate between current checkout data.

If folks have ideas. I'm pretty open to naming here.